src/Npgsql/Util/Statics.cs

```diff
@@ -10,16 +10,19 @@ namespace Npgsql.Util;
 static class Statics
 {
 #if DEBUG
+    internal static bool EnableDiagnostics;
```

Comment:
Probably no need to have DEBUG vs. RELEASE versions - that's only useful in case we want to write tests which turn this on/off, which doesn't seem to be the case here.

Reply:
Sure, just being consistent, doesn't really matter